tools/wpt/update_chromium_revision.py

```diff
@@ -0,0 +1,88 @@
+import requests
+from time import time
+
+from .utils import get
+
+
+PLATFORM_INFO = [
+    ("Win_x64", "chrome-win.zip"),
+    ("Win", "chrome-win.zip"),
```

Comment:
These are the same 4 platforms that Puppeteer checks availability for, but if aligned builds are too rare, dropping win32 should make it about 10x as frequent.

Do you know when the 32-bit windows builds would end up being used? Does it require an old version of windows?

Reply:
It looks like some versions of Windows can come in 32-bits, [up to (rarely) Windows 10](https://www.engadget.com/windows-10-32-bit-oem-173055990.html). This seems really unusual though.

We do technically support installing for 32-bit Windows: 
https://github.com/web-platform-tests/wpt/blob/40e0b5cb8163efe413b448d7682629ac281b13da/tools/wpt/browser.py#L601-L611

This would remove this functionality. If that's okay - the win-32 lookup can definitely be removed.

Reply:
Let's drop it only if that helps get faster updates. It seems like maybe aligned binaries aren't as rare as we first feared.

Reply:
based on the week interval tested earlier (995555-998216)
```
Total viable revisions found: 33
Total viable revisions minus win32: 59
```

So not a major difference. Maybe worth keeping for now in this scenario?

Reply:
Yeah, let's keep it.